src/wp-includes/vars.php

```diff
@@ -146,7 +146,9 @@
  * @return bool
```

Comment:
Maybe add another annotation `@since 6.4.0 Support for the 'HTTP_SEC_CH_UA_MOBILE' header was added.` below the primary `@since` annotation?

Reply:
Done in 1383868e23